.gitignore

```diff
@@ -1 +1,3 @@
 /nbproject/
+*.json
```

Comment:
you are gitignore-ing the mainifest.json? why? @khaleddaifallah-mega ?

Reply:
oh, the IDE added that... I will fix. Thanks

Reply:
resolved.... @lpetrov